doc/source/dev/development_environment.rst

```diff
@@ -76,6 +76,11 @@ Before running the tests, first install the test dependencies::
 
     $ python -m pip install -r test_requirements.txt
 
+.. note:: 
+    For running the benchmarks, install the ``asv`` library by running command in the interactive shell::
+
+        $ python -m pip install asv
```

Comment:
Can you leave out the `.. note::` (it's a lot of visual attention that isn't really necessary) and change this to a line like this right below the use of `test_requirements.txt`?
```
$ python -m pip install asv  # only needed for running benchmarks
```

Reply:
I have made  a commit addressing the requested changes @rgommers